client/README.md

```diff
@@ -0,0 +1,36 @@
+# Name Request Online Frontend
+
+####This project is still very early in its development.  WIP.
+
+## Project setup
+```
+npm install
+```
+
+### Compiles and hot-reloads for development
+```
+npm run serve
+```
+
+### Compiles and minifies for production
+```
+npm run build
+```
+
+### Run your tests
+```
+npm run test
+```
+
+### Lints and fixes files
+```
+npm run lint
+```
+
+### Run your unit tests
+```
+npm run test:unit
```

Comment:
We've done this with this script so it's easy to run unit tests for a specific component:
`"test:unit": "vue-cli-service test:unit --testPathPattern=.",`
(Just replace . with the component file name.)